lib/wings/hydra/pcdm/models/concerns/pcdm_valkyrie_behavior.rb

```diff
@@ -40,6 +41,23 @@ def parent_collection_ids(valkyrie: false)
       end
       # alias member_of_collection_ids child_object_ids # TODO: Cannot alias in this way because member_of_collection_ids method is already defined thru the attribute definition.
 
+      ##
+      # @return [Enumerable<ActiveFedora::Base> | Enumerable<Valkyrie::Resource>] an enumerable over the child collections
+      def child_collections(valkyrie: false)
```

Comment:
I am uncertain, should we use Boolean arguments (rather than introspection `self` or some other Object) when adding methods for `PCDM::Objects` for offering Valkyrie/non-Valkyrie behavior?